test/e2e_node/eviction_test.go

```diff
@@ -83,11 +84,16 @@ var _ = SIGDescribe("InodeEviction", framework.WithSlow(), framework.WithSerial(
 			// Set the eviction threshold to inodesFree - inodesConsumed, so that using inodesConsumed causes an eviction.
 			summary := eventuallyGetSummary(ctx)
 			inodesFree := *summary.Node.Fs.InodesFree
-			if inodesFree <= inodesConsumed {
+			inodesFreeImagefs := *(summary.Node.Runtime.ImageFs.InodesFree)
+
+			if inodesFree <= inodesConsumed && inodesFreeImagefs <= inodesConsumed {
 				e2eskipper.Skipf("Too few inodes free on the host for the InodeEviction test to run")
```

Comment:
e2e tests should be predictable and not be "smart" with complex logic to skip them, this makes the test useless, as it can not give us confidence if the thing that is testing is really being tested, the environment set up has to be determined beforehand

This was like this before, so I don't ask you for changing it, but not for keep building on this

Reply:
thanks for the comments! I'm investigating a bug so forgive me for the sloppy code.